src/angular/directives/dir-beardable.js

```diff
@@ -0,0 +1,67 @@
+'use strict'
```

Comment:
Since this code only ran on the My Widgets page, which has been completely rewritten in React, this file is no longer necessary. I'll make a separate commit to remove it.